src/Transactions/TransactionsList/TransactionsList.js

```diff
@@ -41,6 +41,7 @@ const DEFAULT_SORTING = { [SORTING_PARAMETER]: COL_TRANS_DATE, [SORTING_DIRECTIO
 const resultsPaneTitle = <FormattedMessage id="ui-finance.budget.transactions" />;
 const visibleColumns = [COL_TRANS_DATE, 'transactionType', 'amount', 'fromFundId', 'toFundId', 'source', COL_TAGS];
 const sortableFields = [COL_TRANS_DATE, 'amount', COL_TAGS];
+const nonInteractiveHeaders = ['transactionType', 'fromFundId', 'toFundId', 'source'];
```

Comment:
You already have list of all columns and list of sortables. In the solution you provided dev has manually support these lists, what is error prone. Better solution to pass all columns what is not in sortable list as non interactive